sources/HUBComponentWrapper.m

```diff
@@ -315,13 +346,53 @@ - (BOOL)component:(id<HUBComponentActionPerformer>)component performActionWithId
     return [self.delegate componentWrapper:self performActionWithIdentifier:identifier customData:customData];
 }
 
+#pragma mark - UIGestureRecognizerDelegate
+
+- (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldRecognizeSimultaneouslyWithGestureRecognizer:(UIGestureRecognizer *)otherGestureRecognizer
+{
+    return YES;
+}
+
 #pragma mark - HUBComponentResizeObservingViewDelegate
 
 - (void)resizeObservingViewDidResize:(HUBComponentResizeObservingView *)view
 {
     [(id<HUBComponentViewObserver>)self.component viewDidResize];
 }
 
+#pragma mark - Gesture recognizer handling
+
+- (void)handleGestureRecognizer:(UIGestureRecognizer *)gestureRecognizer
+{
+    switch (gestureRecognizer.state) {
+        case UIGestureRecognizerStatePossible:
+        case UIGestureRecognizerStateChanged:
+            break;
+        case UIGestureRecognizerStateBegan: {
+            self.shouldHighlight = YES;
+            
+            dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(0.2 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
```

Comment:
Oh yeah, should comment on that 😅 It's an attempt to mimic how `UITableView` works, to slightly delay the highlighting in case the user starts scrolling, to avoid "flashing" the highlight 😄